modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/validation/ModelValidationUtils.java

```diff
@@ -19,7 +19,9 @@
 import org.mule.runtime.extension.api.loader.ProblemsReporter;
 import org.mule.runtime.module.extension.internal.loader.java.property.CompileTimeModelProperty;
 
+import java.lang.reflect.Method;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.lang.reflect.Method'. ([squid:UselessImportCheck](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7618&issues=AWhxwX0HotIXnmsUr_SB&open=AWhxwX0HotIXnmsUr_SB)

[ISSUE_KEY]: AWhxwX0HotIXnmsUr_SB